libs/designsystem/src/lib/components/list/list.component.spec.ts

```diff
@@ -141,32 +141,60 @@ describe('ListComponent', () => {
   });
 
   describe('function: ngOnInit', () => {
```

Comment:
Be a good boy scout 🤞 
```suggestion
  describe('function: ngAfterViewInit', () => {
```